### PR TITLE
chore: Support 1.17.13 1.18.10 1.19.3 and add new k8s hotfix

### PR DESCRIPTION
**Pull request by mainred**
**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/k8s_versions.go [hunk 1]
```diff
@@ -189,7 +189,9 @@ func getKubeConfigs() map[string]map[string]string {
 func getVersionOverrides(v string) map[string]string {
 	switch v {
 	case "1.19.1":
-		return map[string]string{"windowszip": "v1.19.1-hotfix.20200923/windowszip/v1.19.1-hotfix.20200923-1int.zip "}
+		return map[string]string{"windowszip": "v1.19.1-hotfix.20200923/windowszip/v1.19.1-hotfix.20200923-1int.zip"}
```
Comment (mainred) [inline pkg/api/k8s_versions.go:192]: @AbelHu , we have an redundant space.

File: pkg/engine/templates_generated.go [hunk 2]
```diff
@@ -41558,7 +41558,7 @@ try
     # to the windows machine, and run the script manually to watch
     # the output.
     if ($true) {
-        Write-Log ".\CustomDataSetupScript.ps1 -MasterIP $MasterIP -KubeDnsServiceIp $KubeDnsServiceIp -MasterFQDNPrefix $MasterFQDNPrefix -Location $Location -AgentKey $AgentKey -AADClientId $AADClientId -AADClientSecret $AADClientSecret -NetworkAPIVersion $NetworkAPIVersion -TargetEnvironment $TargetEnvironment"
+        Write-Log ".\CustomDataSetupScript.ps1 -MasterIP $MasterIP -KubeDnsServiceIp $KubeDnsServiceIp -MasterFQDNPrefix $MasterFQDNPrefix -Location $Location -AADClientId $AADClientId -NetworkAPIVersion $NetworkAPIVersion -TargetEnvironment $TargetEnvironment"
```
Comment (mainred) [inline pkg/engine/templates_generated.go:41561]: We forgot to generate the code in https://github.com/Azure/aks-engine/pull/3918

File: pkg/engine/templates_generated.go [hunk 3]
```diff
@@ -47824,13 +47824,13 @@ var _bintree = &bintree{nil, map[string]*bintree{
...
-			"kubernetesmaster-audit-policy.yaml": {k8sAddonsKubernetesmasterAuditPolicyYaml, map[string]*bintree{}},
+			"antrea.yaml":                                                  {k8sAddonsAntreaYaml, map[string]*bintree{}},
+			"azure-cni-networkmonitor.yaml":                                {k8sAddonsAzureCniNetworkmonitorYaml, map[string]*bintree{}},
+			"azure-policy-deployment.yaml":                                 {k8sAddonsAzurePolicyDeploymentYaml, map[string]*bintree{}},
+			"coredns.yaml":                                                 {k8sAddonsCorednsYaml, map[string]*bintree{}},
+			"dns-autoscaler.yaml":                                          {k8sAddonsDnsAutoscalerYaml, map[string]*bintree{}},
+			"ip-masq-agent.yaml":                                           {k8sAddonsIpMasqAgentYaml, map[string]*bintree{}},
+			"kubernetesmaster-audit-policy.yaml":                           {k8sAddonsKubernetesmasterAuditPolicyYaml, map[string]*bintree{}},
 			"kubernetesmasteraddons-aad-default-admin-group-rbac.yaml":     {k8sAddonsKubernetesmasteraddonsAadDefaultAdminGroupRbacYaml, map[string]*bintree{}},
```
Comment (mainred) [inline pkg/engine/templates_generated.go:47834]: Just a kind remind,  it seems that the code generated in your env is not go-formated @AbelHu

Comment (AbelHu) [reply pkg/engine/templates_generated.go:47834]: interesting. I also run `make generate`

**Review by AbelHu — APPROVED** [review]

**Pull request merged**